templates.rst

```diff
@@ -1634,7 +1629,7 @@ Creating Lazy-Loaded Twig Extensions
 Including the code of the custom filters/functions in the Twig extension class
 is the simplest way to create extensions. However, Twig must initialize all
 extensions before rendering any template, even if the template doesn't use an
-extension.
+extension. Note that if you use attributes, this part is not needed.
```

Comment:
You could instead begin by mentioning the attributes in the very first sentence.

> When using attributes to extend Twig, the services are initialized only when the functions or filters are used to render the template. But in case you use the classic approach by extending the `AbstractExtension` class, Twig initialize all extensions before rendering...